migrations/versions/7f0b46d388a1_.py

```diff
@@ -0,0 +1,80 @@
+"""
+1.4.1 - Move consent_type from partcipant to biospecimen
+
+Revision ID: 7f0b46d388a1
+Revises: c89fea71ca43
+Create Date: 2018-09-20 14:14:31.766224
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import dataservice
+
+# revision identifiers, used by Alembic.
+revision = '7f0b46d388a1'
+down_revision = 'c89fea71ca43'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('biospecimen', sa.Column(
+        'consent_type', sa.Text(), nullable=True))
+    data_upgrades()
+    op.drop_column('participant', 'consent_type')
+    # ### end Alembic commands ###
+
+
+def data_upgrades():
+    """
+    COPY participant.consent_type TO biospecimen.consent_type
+    """
+    connection = op.get_bind()
+    participant = sa.Table(
+        'participant',
+        sa.MetaData(),
+        sa.Column('consent_type', sa.Text(), nullable=True))
+    biospecimen = sa.Table(
+        'biospecimen',
+        sa.MetaData(),
+        sa.Column('consent_type', sa.Text(), nullable=True))
+    results = connection.execute(sa.select([
+        participant.c.consent_type
+    ])).fetchall()
+    for cons in results:
+        connection.execute(biospecimen.update().values(
```

Comment:
What does this do?
Don't you also need to make sure you select the right biospecimen to update? As I understand it, 'results' is just a long list of consent_types. Shouldn't you also need to select the participant's biospecimens and then update each of those?
I think it may be something like `biospecimen.filter(participant_id=participant_id).update().values(consent_type=cons)`